src/main/java/uk/gov/justice/digital/delius/data/api/AddressSummary.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+public class AddressSummary {
```

Comment:
What is the rational for having this class alongside the existing API Address class?

Reply:
The existing address is an offender address, mapped from OFFENDER_ADDRESS & includes offender specific stuff like type, status, assessments (so should really have been named as such).

Reply:
I thought about renaming it but this would be a breaking change for any client using a generated client

Reply:
Perfect. Makes sense